src/styling/about.css

```diff
@@ -0,0 +1,26 @@
+hr {
+  border: 0;
+  clear: both;
+  display: block;
+  background-color: var(--main-yellow);
+  height: 3px;
+  margin-top: 5%;
+  margin-bottom: 5%;
+}
+
+p {
+  font-family: Arial, Helvetica, sans-serif;
+  color: #ffffff;
```

Comment:
I made a var for white as well if you want to change it to that to be consistent!